atomix/cluster/src/main/java/io/atomix/cluster/messaging/impl/AbstractClientConnection.java

```diff
@@ -137,10 +138,12 @@ public void close() {
     Callback(
         final long id,
         final String type,
+        final String receiver,
```

Comment:
Why not just log the receiver at the place where the response is handled. In this case in `FollowerRole:handlePollResponse`. It knows which node was the receiver. So instead of pushing down more information to the messaging layer, we can do it in the caller itself, right?